connectors/http/rest/src/test/resources/requests/success-test-cases.json

```diff
@@ -108,7 +104,6 @@
     "url": "http://localhost:8085/http-endpoint",
     "connectionTimeoutInSeconds": 5,
     "readTimeoutInSeconds": 67,
-    "writeTimeoutInSeconds": 34,
```

Comment:
May you please add an explanation to the PR doc why `writeTimeoutInSeconds` is removed in all of these templates? Thanks 😄